vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue

```diff
@@ -19,14 +19,13 @@
             label="City"
             filled>
           </v-text-field>
-          <v-text-field class="listing-state-entry"
-            solo
-            label="State/Province"
-            filled>
-          </v-text-field>
+          <div class="divider"/>
```

Comment:
just FYI there's a Vuetify divider that you can use called v-divider if you didn't know

Reply:
Awesome! I'll definitely start using it! 👍